src/test/java/com/iexec/common/docker/client/DockerNetworkTests.java

```diff
@@ -41,7 +41,7 @@ void shouldNotCreateNetworkSinceEmptyName() {
     @Test
     void shouldReturnExistingNetworkIdWenNetworkIsAlreadyPresent() {
         String networkName = getRandomString();
-        String networkId = dockerClientInstance.createNetwork(networkName);
+        dockerClientInstance.createNetwork(networkName);
```

Comment:
I have a rework on this test method in a next PR.
It will be modified again with a check on `networkId` to have a check which makes sense with the test method name.

Reply:
Wasn't sure about it, no worry to re-add it later, thanks for the info :ok_hand: